packages/tasks/src/model-libraries.ts

```diff
@@ -553,6 +553,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	birefnet : {
+		prettyLabel : "BiRefNet",
+		repoName : "BiRefNet",
+		repoUrl : "https://github.com/ZhengPeng7/BiRefNet",
+		docsUrl : "https://www.birefnet.top",
+		filter: false,
+		countDownloads: { term: { path: "model.safetensors" } },
```

Comment:
The pattern to add a download count has changed recently. Could you update it accordingly (it should be easier now without having to type an ES query)

Reply:
I'm going to remove the countDownloads parameter since the downloads are already working for https://huggingface.co/ZhengPeng7/birefnet.

Reply:
Ah yes, it's based on the `config.json` file thanks to the mixin :)